src/client/manager/provider.js

```diff
@@ -4,6 +4,7 @@ import React from 'react';
 import { Provider } from '@kadira/storybook-ui';
 import addons from '@kadira/storybook-addons';
 import createChannel from '@kadira/storybook-channel-pagebus';
+import createDatabase from '@kadira/storybook-database-local';
```

Comment:
Move it to the same module. We can import the middleware like this:

~~~js
import middleware from '@kadira/storybook-database-local/dist/middleware'
~~~